index.html

```diff
@@ -57,6 +60,164 @@ <h1 id="primary-text">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
```

Comment:
- [ ] I would highly suggest you to not do nesting of a single element `button` inside a `div `and then inside the `button `using a `p `tag as its semantically incorrect also it's making your code really complicated and difficult to read.